src/agent/commands/results.publish.ts

```diff
@@ -37,20 +37,10 @@ export class ResultsPublishCommand implements cm.IAsyncCommand {
         var publishRunAttachments: boolean = (this.command.properties['publishRunAttachments'] === "true");
         var resultFilesPath = this.command.properties['resultFiles'];
         var mergeResults: boolean = (this.command.properties['mergeResults'] === 'true');
+        var fileNumber: string = this.command.properties['fileNumber'];
```

Comment:
It's not missed. It's no longer require by new way of handling this task. it's only required for old task compat